keylime/common/algorithms.py

```diff
@@ -12,13 +12,34 @@ def is_accepted(algorithm: str, accepted: List[Any]) -> bool:
     return algorithm in accepted
 
 
+def _hashit(algorithm: str, data: bytes) -> bytes:
```

Comment:
I do not think that this should be extracted from the class, and the main reason is that it is making worse a problem that the old class has: the list of supported hashes: is it replicated all over the code

Reply:
I had to extract it to be able to call this function from the constructor since I could neither call a @staticfunction nor a @classsfunction in the constructor. Nobody should use this function directly.

Reply:
Actually, that was wrong. I can revert the extraction. PR soon.

What does not work is using Hash.SHA1 etc when calling the private function from the constructor:

```
  File "/home/stefanb/keylime/keylime/common/algorithms.py", line 36, in __hash
    if self.value == Hash.SHA1:
```

Reply:
I will try to move a bit in this direction:  https://github.com/keylime/keylime/pull/1191

The idea is that to add a new hash algorithm, only the enum needs to be extended